packages/graphback-runtime-mongodb/tests/MongoDataProviderTest.ts

```diff
@@ -213,4 +213,59 @@ describe('MongoDBDataProvider Basic CRUD', () => {
       expect(todos[t].text).toEqual(`todo${5 - t}`);
     }
   });
+
+  test('createdAt', async () => {
+    context = await createTestingContext(`
+    """@model"""
+    type Note {
+      id: ID!
+      text: String
+      """
+      @createdAt
+      """
+      created: String
+      """
+      @updatedAt
+      """
+      lastModified: String
+    }
+    `);
+    const cDate = new Date(2020, 5, 26, 18, 29, 23);
+    advanceTo(cDate);
+
+    const res = await context.providers.Note.create({ text: 'asdf' });
+    expect(res.created).toEqual(cDate.getTime());
+  })
+
+  test('updatedAt', async () => {
+    context = await createTestingContext(`
+    """@model"""
+    type Note {
+      id: ID!
+      text: String
+      """
+      @createdAt
```

Comment:
I thought we agreed with different format ( `@versioned` or something like that

Reply:
Haven't updated tests yet